app/models/item.rb

```diff
@@ -6,9 +6,12 @@ class Item < ActiveRecord::Base
 
   has_attached_file :image, styles: {
       thumb: '100x100>',
-      square: '200x200#',
+      square: '200x200',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.